x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/tour.tsx

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { EuiText, EuiTourStep } from '@elastic/eui';
-import React, { useCallback, useEffect, useState } from 'react';
-import { RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY } from '../../../../../common/constants';
-import { useKibana } from '../../../../common/lib/kibana';
-import * as i18n from './translations';
-
-export interface Props {
-  children: React.ReactElement;
-}
-
-export const RulesPageTourComponent: React.FC<Props> = ({ children }) => {
-  const tourConfig = {
-    currentTourStep: 1,
-    isTourActive: true,
-    tourPopoverWidth: 300,
-  };
-
-  const { storage } = useKibana().services;
-
-  const [tourState, setTourState] = useState(() => {
-    const restoredTourState = storage.get(RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY);
```

Comment:
Could you please delete the constant `RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY` as well? It's a duplicate of `NEW_FEATURES_TOUR_STORAGE_KEYS.RULE_MANAGEMENT_PAGE`.